src/store/ChartSettingStore.js

```diff
@@ -74,6 +74,11 @@ export default class ChartSettingStore {
     @observable language = '';
     @observable position = '';
     @observable theme = '';
+    defaultLanguage = {
```

Comment:
minor tweak: I think this can be changed to just `defaultLanguage = languages[0]`.

Also `languages` doesn't need to be `@observable`